sn_node/src/node/messaging/mod.rs

```diff
@@ -36,9 +37,9 @@ use std::collections::BTreeSet;
 #[derive(Debug, Clone)]
 #[allow(clippy::large_enum_variant)]
 pub(crate) enum OutgoingMsg {
-    System(SystemMsg),
-    Service(ServiceMsg),
-    DstAggregated((BlsShareAuth, Bytes)),
+    Node2Node(Node2NodeMsg),
```

Comment:
I'd reduce this to `Node` as well.
```suggestion
    Node(Node2NodeMsg),
```